policies/k8sallowedrepos.rego

```diff
@@ -0,0 +1,32 @@
+package k8sallowedrepos
```

Comment:
rename all of these packages:

- strip the `k8s` part
- put dashes between words, so in this case `allowed-repos`
- rename the files to be the same